workitems/pyproject.toml

```diff
@@ -23,7 +23,7 @@ mypy_path = "src:tests"
 
 [tool.poetry.dependencies]
 python = "^3.9"
-robocorp-tasks = ">=1,<3"
+robocorp-tasks = "^3.0.0"
```

Comment:
Strange older version pin, but anyway, we can't build now the **meta** without this upgrade, which requires a release first.

Reply:
As we discussed, we should probably keep the same approach with `">=1,<4"` -- nothing changed should've broken `workitems`, so, we don't really need to specify the latest as the minimum version.